### DISCUSSION TITLE
Tab type card with rounded corners

### DISCUSSION THREAD

**Discussion by vgs18** [ant-design/ant-design#41770] [General]
### What problem does this feature solve?

I want to have a tab with the corners rounded for the content and tabs both.
I was able to add the border to content but the top right corner border radius is not working as the border bottom of tabs cannot be rounded to bottom. Is there a way we can do that?

<img width="654" alt="image" src="https://user-images.githubusercontent.com/95902397/230387684-fb854373-086e-4b4a-a59c-ba274128a453.png">

### What does the proposed API look like?

The tabs with all the outer corners rounded including the content

**Comment by Wxh16144** [comment]
Is [this example](https://ant.design/components/tabs-cn#components-tabs-demo-card-top) the desired effect? You can customize the style to override it.

**Comment by vgs18** [comment]
@Wxh16144  Thanks for the quick reply and the example. I appreciate the time and effort.
The example here adds the border around the tab space including the empty space where tabs are not present. I want the borders around the tabs and to the content and not including the empty space to the right of the tabs.
I was able to add the border for content, but this will also add the border below the selected tab which is not required. Is there a way we can add the border to all except the border bottom of selected tab and the space where tabs are not present.

**Comment by Wxh16144** [comment]
Is that right?
![image](https://user-images.githubusercontent.com/32004925/230622551-9682ae93-7786-48fd-b6be-b288c23d90c2.png)

**Comment by vgs18** [comment]
@Wxh16144 more near to the designs but exactly  I am looking for this 
![2023_04_07_0zb_Kleki](https://user-images.githubusercontent.com/95902397/230637326-cd8540c1-da6c-41ad-9946-4eb914ed09f3.png)

All that I am not able to do is border radius on the top right.

**Comment by vgs18** [comment]
@Wxh16144 Any updates on this?
